en/option/series/treemap.md

```diff
@@ -169,6 +169,9 @@ When is no content in breadcrumb, this minimal width need to be set up.
 ) }}
 
 ### emphasis(*)
+{{ use: partial-version(
+    version = "5.4.0"
+) }}
```

Comment:
It seems we also need to correct the default values of the following `itemStyle` object.

https://github.com/apache/echarts-doc/blob/master/en/option/series/treemap.md?plain=1#L181-L188
https://github.com/apache/echarts-doc/blob/master/en/option/series/treemap.md?plain=1#L155-L161

https://github.com/apache/echarts/blob/release/src/chart/treemap/TreemapSeries.ts#L267-L277